src/css/units.css

```diff
@@ -3,5 +3,5 @@ $space: 0.5rem;
 $sprites-per-row: 5;
 
 $menu-bar-height: 3rem;
-$sprite-info-height: 5rem;
+$sprite-info-height: 5.25rem; /* TODO: SpriteInfo isn't explicitly set to this height yet */
```

Comment:
should I do a pass to update the convention? Added a lot of these in the last few weeks